app/src/main/java/org/schabi/newpipe/local/feed/FeedViewModel.kt

```diff
@@ -32,17 +32,23 @@ class FeedViewModel(
     private val application: Application,
     groupId: Long = FeedGroupEntity.GROUP_ALL_ID,
     initialShowPlayedItems: Boolean = true,
+    initialShowPartiallyPlayedItems: Boolean = true,
     initialShowFutureItems: Boolean = true
```

Comment:
While you are at it, could you remove the default values? I think they are always overwritten and may create confusion